src/PlusDataCollection/InfraredSeekCam/vtkInfraredSeekCam.h

```diff
@@ -47,6 +47,10 @@ class vtkPlusDataCollectionExport vtkInfraredSeekCam : public vtkPlusDevice
   /*! Verify the device is correctly configured */
   virtual PlusStatus NotifyConfigured();
 
+  bool readBinaryFile(const std::string& filename, cv::Mat& temp) const;
```

Comment:
Minor changes, but could you apply the code formatting guidelines?

readBinaryFile -> ReadBinaryFile
readImage -> ReadImage
Settemp -> SetTemp